src/whos_there/senders/slack.py

```diff
@@ -6,25 +6,37 @@
 
 
 class SlackSender(Sender):
-    def __init__(self, webhook_url: str, channel: str, user_mentions: list[str] | None = None) -> None:
+    def __init__(
+        self,
+        webhook_url: str,
+        channel: str,
+        user_mentions: list[str] | None = None,
+        sender_username: str = "Who's There notification",
```

Comment:
```suggestion
        sender_username: str = "Kock Knock",
```
I would like to keep the joke as a default argument